### Notebook Diffs vs origin/main
nbdiff /dev/null new-nb.ipynb
--- /dev/null  2025-07-30 12:04:44.623812
+++ new-nb.ipynb  2025-07-30 12:04:34.238326
## added /cells:
+  code cell:
+    source:
+      what happens now?




